database/README.md

```diff
@@ -0,0 +1,40 @@
+## Description
+
+This Folder is responsible for defining and creating the tables in our database.
+
+
+## Requirements
+There are no requirements for this folder.
```

Comment:
Include `brew install postgresql@14` as a requirement

Reply:
I think Shindy is covering this in the main README